core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -592,6 +591,25 @@ protected function createExtract()
 		$this->arrUniqueFields = array_unique($this->arrUniqueFields);
 		$this->blnIsDbTable = true;
 	}
+
+	private function getDatabaseSqlFiles(): array
+	{
+		if (null !== self::$arrDatabaseSqlFiles)
+		{
+			return self::$arrDatabaseSqlFiles;
+		}
+
+		try
+		{
+			$files = System::getContainer()->get('contao.resource_locator')->locate('config/database.sql', null, false);
+		}
+		catch (FileLocatorFileNotFoundException $e)
```

Comment:
this now catches a specific exception instead of `\InvalidArgumentException`, is that intentional?

Reply:
Yes, that's the one that Symfony file locator is throwing if a file does not exist.